composer.json

```diff
@@ -47,6 +47,7 @@
     },
     "conflict": {
         "simplethings/entity-audit-bundle": ">=2.0",
+        "sonata-project/admin-bundle": "4.0.0-alpha",
```

Comment:
this packagist version needs to be removed @rande

Reply:
Not sure if we need this. An alpha version is always risky

Reply:
Yes, but that was tagged accidentaly, the correct one is 4.0.0-alpha-1

Reply:
I know, but we should remove the conflict once the tagged release was removed from packagist.

Reply:
Agree, once the tagged release is removed, I will change this file again.

Reply:
The conflict is not needed.
The release `4.0.0-alpha` has the same code than the `4.0.0-alpha-1` one.

Reply:
Removed